apps/balancer-tools/src/app/apr/(utils)/calculatePoolStats.ts

```diff
@@ -115,7 +112,17 @@ function calculateRoundAPR(
   votingShare: number,
   tvl: number,
   balPriceUSD: number,
-): number {
+) {
   const emissions = balEmissions.weekly(round.endDate.getTime() / 1000);
-  return (WEEKS_IN_YEAR * (emissions * votingShare * balPriceUSD)) / tvl;
+  const vebalAPR =
+    balPriceUSD && tvl && votingShare
+      ? ((WEEKS_IN_YEAR * (emissions * votingShare * balPriceUSD)) / tvl) * 100
+      : -1;
+
```

Comment:
emit a warning if this result is -1, preferably we'd be warned on Sentry when this happens with the data to reproduce it.